lib/theme-and-appearance.zsh

```diff
@@ -17,6 +17,8 @@ then
     # coreutils, so prefer it to "gls".
     gls --color -d . &>/dev/null 2>&1 && alias ls='gls --color=tty'
     colorls -G -d . &>/dev/null 2>&1 && alias ls='colorls -G'
+  elif [[ "$(uname -s)" == "OS400" ]]; then
```

Comment:
Could this be done with a generic test of whether `ls -G` is supported, instead of adding an OS-specific test?